verilog/formatting/formatter.cc

```diff
@@ -379,6 +396,45 @@ static void DeterminePartitionExpansion(
         node_view.Expand();
       }
     }
+
```

Comment:
Is there a break missing here @wtatarski (or: ABSL_FALLTHROUGH_INTENDED) ?
(also a test should trigger if the uintended situation is happening)

Please add a follow-up CL to clarify this part of the code (and possibly add a test that makes sure that this is caught)